demos/demo23/demo23.nim

```diff
@@ -0,0 +1,13 @@
+import
+  nimgame2 / [
+    nimgame, settings
+  ],
+  mainscene
+
+
+game = newGame()
+settings.updateInterval = 1000
+if game.init(400, 400, "TextEntity module test"):
```

Comment:
Could you change the resolution to 640x480 like all other demos? Plus, engine's info-panel obscures your objects in the top-left corner right now.